File: python/prism_sdk/multimodal_resource_discovery_workflow_fabric.py
"""Worldgen P05 AFA-worldgen-P05-F14 product wrapper."""
from .worldgen_resource_workflow_support import ResourceWorkflowRequest, ResourceWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P05-F14"
CONTRACT_VERSION="worldgen-multimodal-resource-workflow/1.0"
def worldgen_multimodal_resource_discovery_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="ResourceWorkflowRequest1@1", scale="multimodal multi-study", autonomy_tier="A1")
def schedule_worldgen_multimodal_resource_discovery_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="multimodal multi-study", require_approval=False, require_federation=False)
__all__=["ResourceWorkflowRequest","ResourceWorkflowReceipt","worldgen_multimodal_resource_discovery_workflow_fabric_manifest","schedule_worldgen_multimodal_resource_discovery_workflow"]

File: crates/worldgen/src/multimodal_resource_discovery_workflow_fabric.rs
//! Worldgen P05 AFA-worldgen-P05-F14 multimodal workflow_fabric.
use super::resource_workflow_support::{self,ResourceWorkflowRequest,ResourceWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P05-F14"; pub const CONTRACT_VERSION:&str="worldgen-multimodal-resource-workflow/1.0";
pub fn worldgen_multimodal_resource_discovery_workflow_fabric_manifest()->serde_json::Value{resource_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"ResourceWorkflowRequest1@1","multimodal multi-study","A1")}
pub fn schedule_worldgen_multimodal_resource_discovery_workflow(request:&ResourceWorkflowRequest)->Result<ResourceWorkflowReceipt,resource_workflow_support::ResourceWorkflowError>{resource_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"multimodal multi-study",false,false)}
pub use resource_workflow_support::{ResourceWorkflowError,ResourceWorkflowReceipt as WorldgenmultimodalResourceworkflowfabricReceipt,ResourceWorkflowRequest as WorldgenmultimodalResourceworkflowfabricRequest};
